enterprise/admin/forms.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Custom admin forms.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import forms
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.utils.translation import ugettext as _
+
+from enterprise.models import EnterpriseCustomerUser
+
+
+class ManageLearnersForm(forms.Form):
+    """
+    Form to manage learner additions.
+    """
+    email = forms.CharField(label=_("Email or Username"))
+
+    def clean_email(self):
+        """
+        Clean email form field
+        :return: str email
```

Comment:
@e-kolpakov nice suggestion. Its upto us which pattern we follow going forward, I am fine with whatever pattern we choose. Only thing I would say is that we have a very wide community of open source users so we have to be consistent what ever we do.